### PR TITLE
Fix list datasets

### PR DESCRIPTION
**Pull request by kmax12**
Fix bug where `GridStatusClient.list_datasets` would error when `return_list=False`

### CODE REVIEW

File: gridstatusio/gs_client.py [hunk 1]
```diff
@@ -283,9 +286,9 @@ def get_dataset(
                 "page": page,
                 "page_size": page_size,
                 "resample_frequency": resample,
-                "resample_by": ",".join(resample_by)
-                if resample_by is not None
-                else None,
+                "resample_by": (
```
Comment (WillKoehrsen) [inline gridstatusio/gs_client.py:289]: Is this change related to #30?

**Review by WillKoehrsen — APPROVED** [review]
This solves the problem!

File: gridstatusio/gs_client.py (hunk 1 above)
Comment (kmax12) [reply gridstatusio/gs_client.py:289]: i think this just a formatting change
